db/migrate/20230328180336_add_foreign_key_to_likes.rb

```diff
@@ -0,0 +1,8 @@
+class AddForeignKeyToLikes < ActiveRecord::Migration[7.0]
+  def change
+  add_foreign_key :likes, :posts, column: :post_id
+  add_foreign_key :likes, :users, column: :user_id
```

Comment:
- [ ] Please kindly ensure that the foreign  key references the `author_id` as specified in the schema diagram